en/identity-server/7.0.0/docs/assets/code-samples/illustration.svg

```diff
@@ -0,0 +1 @@
351,12.63351,0,0,1,8.18076,6.80183,13.25027,13.25027,0,0,1-1.84274,14.40983,14.17281,14.17281,0,0,1-14.56665,4.84008Z" transform="translate(-223.84174 -181.89597)" fill="#63ff97"/><polygon points="492.621 518.459 476.984 518.458 469.545 458.145 492.623 458.146 492.621 518.459" fill="#ffb8b8"/><polygon points="614.374 511.69 598.913 514.027 582.541 455.503 602.81 452.054 614.374 511.69" fill="#ffb8b8"/><path d="M761.70816,441.11609l-54.08209-9.20546-1.92351,3.96487A183.65217,183.65217,0,0,0,687.706,504.08793c-1.30584,19.99608-2.0741,41.11515-2.0993,63.25389-.03905,34.301,1.71127,66.12865,4.47813,95.08942l29.68206-.18583a701.05428,701.05428,0,0,1,3.49468-92.11432c2.27179-21.81641,5.43055-42.19545,9.08269-61A419.62473,419.62473,0,0,0,760.887,589.25522c10.37279,21.9358,31.70832,47.29156,42.081,69.22736q15.94307-1.91322,31.88615-3.82638c-8.37624-22.15114-23.319-54.49822-31.69519-76.64935-11.9764-31.67166-17.38631-53.14723-29.36271-84.81889a97.61477,97.61477,0,0,0,.39857-17.936c-.558-9.96443-2.63061-21.92172-8.689-29.89328Z" transform="translate(-223.84174 -181.89597)" fill="#2f2e41"/><path d="M770.06329,424.60665c2.70081,5.04847-3.71356,15.5073-8.35513,16.50944l-53.4815-17.28777,26.929-78.03259,13.54851,2.46246,21.66846,3.65071,7.686,6.98632a13.513,13.513,0,0,1,3.727,14.26283Z" transform="translate(-223.84174 -181.89597)" fill="#63ff97"/><path d="M733.24663,394.45l-15.12956-9.65517-19.87785,19.96339L660.71691,439.6668a10.27076,10.27076,0,1,0,7.97721,11.13774l41.38042-31.01482Z" transform="translate(-223.84174 -181.89597)" fill="#ffb8b8"/><path d="M724.12187,409.51008,713.16117,385.079l.24-.30344c.151-.19051,15.2663-19.24628,27.40135-30.41071a12.63382,12.63382,0,0,1,10.03144-3.54359,13.25745,13.25745,0,0,1,11.43348,8.96171,14.1712,14.1712,0,0,1-3.30149,14.98926Z" transform="translate(-223.84174 -181.89597)" fill="#63ff97"/><circle cx="534.37189" cy="134.89961" r="23.78567" fill="#ffb8b8"/><path d="M777.42309,351.26194l-.091-.023-34.56543-8.64529A7.90147,7.90147,0,0,1,737.91,330.86036l1.14169-1.89964a12.95982,12.95982,0,0,0,.53039-12.37883,13.64408,13.64408,0,0,0-7.84464-6.7007l-.11462-.05506-.72592-.45173a5.66354,5.66354,0,0,1-2.99807-3.46441,5.84347,5.84347,0,0,1,.572-4.7123,28.59783,28.59783,0,0,1,13.0643-11.5175,29.49379,29.49379,0,0,1,21.67756-.65119,3.84942,3.84942,0,0,1,3.53969-.89953c17.90076,4.33528,19.374,16.35059,15.83762,35.638l4.18584,16.88211a8.45854,8.45854,0,0,1-7.54574,10.4702Zm-14.40152-62.043-.00224.00224Zm.43151-.10563.00337.00112Z" transform="translate(-223.84174 -181.89597)" fill="#2f2e41"/><path d="M720.2731,715.99017h-8.92427l-1.59294-8.42557-4.07977,8.42557h-23.6693a5.32064,5.32064,0,0,1-3.02337-9.699L697.88506,693.237v-8.518l19.88119,1.18664Z" transform="translate(-223.84174 -181.89597)" fill="#2f2e41"/><path d="M840.70475,715.99017h-8.92427l-1.59294-8.42557-4.07977,8.42557h-23.6693a5.32064,5.32064,0,0,1-3.02337-9.699L818.31671,693.237v-8.518l19.88119,1.18664Z" transform="translate(-223.84174 -181.89597)" fill="#2f2e41"/></svg>
```

Comment:
We may need to add the license header

Reply:
Also add newline

Reply:
fixed with https://github.com/wso2/docs-is/pull/5157/commits/34741ff481d9b2fe5a4063b7e8a96a63ee6b5d1b